### PR TITLE
Certora review

### PR DESCRIPTION
**Pull request by nisnislevi**
## Solution

### CODE REVIEW

File: certora/specs/ccip.spec [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  uint256 curr_bridge_after = getCurrentBridgedAmount();
+
+  assert 
+    curr_bridge_after < curr_bridge_before =>
+    f.selector==sig:releaseOrMint(bytes memory,address,uint256,uint64,bytes memory).selector;
+}
+
+
+/* ==============================================================================
+   rule: only_releaseOrMint_can_deccrease_currentBridged
```
Comment (miguelmtzinf) [inline certora/specs/ccip.spec:114]: seems incorrect

Comment (nisnislevi) [reply certora/specs/ccip.spec:114]: Please elaborate. Why is it incorrect ? which function can also decrease s_currentBridged ?

Comment (miguelmtzinf) [reply certora/specs/ccip.spec:114]: It seems this comment refers to the code below, which is about setting the bridge limit

Comment (nisnislevi) [reply certora/specs/ccip.spec:114]: Can you please elaborate what is wrong there ?

The rule merely says that: Only the bridgeLimitAdmin or the owner can call setBridgeLimit(). This is is accordance with the solidity code (and the documentation in the solidity code):

if (msg.sender != s_bridgeLimitAdmin && msg.sender != owner()) revert Unauthorized(msg.sender);

Comment (miguelmtzinf) [reply certora/specs/ccip.spec:114]: It's just the name of the rule in the comment what is wrong (L114) 
```suggestion
   rule: only_bridgeLimitAdmin_or_owner_can_call_setBridgeLimit
```

Comment (nisnislevi) [reply certora/specs/ccip.spec:114]: Or did you meant that the comment is incorrect ?

**Pull request closed without merging**